RNative2/components/UI/Title.js

```diff
@@ -1,19 +1,21 @@
-import { StyleSheet, Text } from "react-native";
-import Colors from "../../constants/colors";
+import { StyleSheet, Text, Platform } from "react-native";
 
 const Title = ({ children }) => {
   return <Text style={styles.title}>{children}</Text>;
 };
 
 const styles = StyleSheet.create({
   title: {
+    fontFamily: "open-sans-bold",
     fontSize: 24,
-    fontWeight: 700,
     color: "#ffffff",
     textAlign: "center",
-    borderWidth: 2,
+    // borderWidth: Platform.OS === "android" ? 2 : 0,
```

Comment:
Please remove the commented out code.

Reply:
Kept the comments as per our call - keeping it for learning purposes